lib/utilities.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+from dateutil.parser import parse
+import pytz
+
+def toUTC(suspectedDate, localTimeZone='UTC'):
```

Comment:
A couple notes here:

1. I decided to use UTC as the default timezone, I couldn't figure out how to grab the local timezone using tzlocal that didn't cause stuff to break, but the assumption I had was to assume we wanted UTC? I will probably circle back to this to see if I can use tzlocal.

2. I didn't move any of the cron scripts or anything that have this method redefined yet, I'm more or less looking for reviewers to see if this is how we want to tackle this.